packages/os/os.spec

```
@@ -45,6 +45,7 @@ Source114: bootstrap-containers@.service
 Source115: link-kernel-modules.service
 Source116: load-kernel-modules.service
 Source117: cfsignal.service
+Source118: network-config.service
```

nit: to disambiguate from a service that configures network devices:
```suggestion
Source118: generate-network-config.service
```